moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -232,7 +225,7 @@ const ProjectActivityLog = () => {
                         width="5%"
                         className={classes.tableCell}
                       >
-                        {formatDate(change.created_at)}
+                        {formatTimeStampZType(change.created_at)}
```

Comment:
Fixes date shown in Activity Log https://github.com/cityofaustin/atd-data-tech/issues/7121